publications/joss/paper.bib

```diff
@@ -531,25 +520,6 @@ @article{batalha2019picaso
   publisher={IOP Publishing}
 }
 
-@article{bean2018transiting,
-  title={The transiting exoplanet community early release science program for JWST},
-  author={Bean, Jacob L and Stevenson, Kevin B and Batalha, Natalie M and Berta-Thompson, Zachory and Kreidberg, Laura and Crouzet, Nicolas and Benneke, Bj{\"o}rn and Line, Michael R and Sing, David K and Wakeford, Hannah R and others},
-  journal={Publications of the Astronomical Society of the Pacific},
-  volume={130},
-  number={993},
-  pages={114402},
-  year={2018},
-  publisher={IOP Publishing}
-}
-
-@article{MacDonald2023POSEIDON,
```

Comment:
This bibliography entry is a duplicate, so I have remove this and changed the accompanying citation in the paper to the other entry.